.github/workflows/docker-build.yaml

```diff
@@ -69,10 +71,14 @@ jobs:
               docker/Dockerfile.base
             sed -i 's/balenalib\/raspberrypi3/balenalib\/raspberry-pi2/' \
               docker/Dockerfile.viewer
+            sed -i 's/\bBALENA_ARCH\b/rpi/' \
```

Comment:
Can we make this consistent with the structure in the other `Dockerfiles` instead?

Reply:
I'm not sure if I understood this very well.
Did you mean like making `armv7fh` the default on the `Dockerfile.wifi-connect`, then just replacing the arch with `rpi` (using `sed`) for Raspberry Pi 1 and 2 devices?

![image](https://user-images.githubusercontent.com/10234135/210421991-3ebd35d2-3c50-4fe2-b4f7-b19139372ca6.png)

![image](https://user-images.githubusercontent.com/10234135/210422097-7da24fca-cfff-44dc-be43-8a9399a2e911.png)

Or, is it necessary to go an extra step by using a different related image?

Reply:
Rework this to use `Dockerfile.base` as the base image instead. Otherwise we need to pull down a whole different image (`balenalib/armvhf-python:3.7-buster`) that will require a lot of more download and disk space.

After you've done this, rework it to use the same structure that we use for all the other images in [docker-build.yaml](https://github.com/Screenly/Anthias/blob/master/.github/workflows/docker-build.yaml). There's no point in reinventing the wheel here.